bin/tag-release-prerelease.js

```diff
@@ -12,6 +12,9 @@ commander.parse(process.argv);
 let options = {};
 const { verbose, maxbuffer, args } = commander;
 const prerelease = args.length ? args[0] : true;
-options = extend({}, { prerelease, verbose, maxbuffer, workflow });
+options = extend(
```

Comment:
I don't think the `extend` is needed here. You should just be able to do something like...

```
options = { prerelease, verbose, maxbuffer, workflow, command: "prerelease" };
```